rubix-hadoop2/src/main/java/com/qubole/rubix/hadoop2/Hadoop2ClusterManager.java

```diff
@@ -183,24 +179,26 @@ public void setNode(List<Elements> node)
     public static class Elements
     {
         /*
-        rack 	         string
-        state 	         string
-        id 	             string
-        nodeHostName 	 string
+        rack             string
+        state            string
+        id               string
+        nodeHostName     string
         nodeHTTPAddress  string
-        healthStatus 	 string
-        healthReport 	 string
+        healthStatus     string
+        healthReport     string
         lastHealthUpdate long
-        usedMemoryMB 	 long
-        availMemoryMB 	 long
-        numContainers 	 int
+        usedMemoryMB     long
+        availMemoryMB    long
+        numContainers    int
         */
 
         String nodeHostName;
         String state;
 
-        String getState() { return state; }
+        String getState()
+        { return state; }
```

Comment:
this seems invalid, shouldn't return be on next line and '}' and in line after that?